tests/e2e/autoscaling/00-create-vm.yaml

```diff
@@ -0,0 +1,146 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- ../../../deploy/autoscale-scheduler.yaml
+- ../../../deploy/autoscaler-agent.yaml
+unitTest: false
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: example
+spec:
+  ports:
+  - name: postgres
+    port: 5432
+    protocol: TCP
+    targetPort: postgres
+    nodePort: 30432
+  - name: pooler
+    port: 6432
+    protocol: TCP
+    targetPort: postgres
+    nodePort: 30632
+  - name: host-metrics
+    port: 9100
+    protocol: TCP
+    targetPort: host-metrics
+    nodePort: 30100
+  - name: metrics
+    port: 9187
+    protocol: TCP
+    targetPort: metrics
+    nodePort: 30187
+  - name: informant
+    port: 10301
+    protocol: TCP
+    targetPort: informant
+    nodePort: 30301
+  type: NodePort
+  selector:
+    vm.neon.tech/name: example
+---
+apiVersion: vm.neon.tech/v1
+kind: VirtualMachine
+metadata:
+  name: example
+spec:
+  schedulerName: autoscale-scheduler
+  guest:
+    cpus:
+      min: 1
+      max: 4
+      use: 1
+    memorySlotSize: 1Gi
+    memorySlots:
+      min: 1
+      max: 4
+      use: 1
+    rootDisk:
+      image: vm-example:dev
+      size: 8Gi
+    args:
+      - -c
+      - 'config_file=/etc/postgresql/postgresql.conf'
+    env:
+      # for testing only - allows login without password
+      - name: POSTGRES_HOST_AUTH_METHOD
+        value: trust
+    ports:
+      - name: postgres
+        port: 5432
+      - name: pooler
+        port: 6432
+      - name: host-metrics
+        port: 9100
+      - name: metrics
+        port: 9187
+      - name: informant
+        port: 10301
+  extraNetwork:
+      enable: true
+  disks:
+    - name: pgdata
+      mountPath: /var/lib/postgresql
+      emptyDisk:
+        size: 16Gi
```

Comment:
Does this work? I'd be concerned about issues around disk migration

Reply:
It works (as in tests for neonvm). Do you think we need to change it to let's wait until something breaks? 😄

Reply:
Ah ok, yeah should be fine to leave it - plus easy enough to remove later if things break :sweat_smile: